boards/nxp/mcx_nxxx_evk/board.cmake

```diff
@@ -27,6 +27,8 @@ elseif(CONFIG_SOC_MCXN947_CPU1)
   board_runner_args(jlink "--device=MCXN947_M33_1" "--reset-after-load")
   board_runner_args(linkserver  "--device=MCXN947:MCX-N9XX-EVK")
   board_runner_args(linkserver  "--core=cm33_core1")
+elseif(CONFIG_SOC_MCXN547_CPU0)
+  board_runner_args(linkserver  "--device=MCXN547:MCX-N5XX-EVK")
```

Comment:
we need to add JLink support for the MCXN547 device

Reply:
okay, that can be done in a follow up PR